Steps4Impact/App/AppController.swift

```diff
@@ -69,8 +69,8 @@ class AppController {
 
     // Select Default View
     // TODO(samisuteria) add check for akf profile?
-    print("Facebook Id: \(Facebook.id)")
-    if Facebook.id.isEmpty {
+    let currentUser = Auth.auth().currentUser
```

Comment:
Is this copy to avoid some sort of reference count issue?  Seems like its a single use variable, why not inline it?

Reply:
Good call - I'll update this to use the new `User.id` helper instead